frontend/package.json

```diff
@@ -38,5 +39,8 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8080"
+  "proxy": "http://localhost:8080",
+  "devDependencies": {
+    "react-router-dom": "^5.2.0"
+  }
```

Comment:
```suggestion
```
you already have this dependency. This should be part of the normal dependencies.
But you can add prettier here